### ISSUE TITLE
turning on debug is not working

### ISSUE THREAD

**Issue by hackruu** [lprhodes/homebridge-broadlink-rm#320]
```мар 21 00:41:20 homekit homebridge[11645]: /usr/lib/node_modules/homebridge-broadlink-rm/accessories/accessory.js:12
мар 21 00:41:20 homekit homebridge[11645]:     if (config.debug) this.debug = true
мар 21 00:41:20 homekit homebridge[11645]:                       ^
мар 21 00:41:20 homekit homebridge[11645]: ReferenceError: Must call super constructor in derived class before accessing 'this' or returning from derived constructor
мар 21 00:41:20 homekit homebridge[11645]:     at new BroadlinkRMAccessory (/usr/lib/node_modules/homebridge-broadlink-rm/accessories/accessory.js:12:23)
мар 21 00:41:20 homekit homebridge[11645]:     at new SwitchAccessory (/usr/lib/node_modules/homebridge-broadlink-rm/accessories/switch.js:10:5)
мар 21 00:41:20 homekit homebridge[11645]:     at config.accessories.forEach (/usr/lib/node_modules/homebridge-broadlink-rm/platform.js:79:32)
мар 21 00:41:20 homekit homebridge[11645]:     at Array.forEach (<anonymous>:null:null)
мар 21 00:41:20 homekit homebridge[11645]:     at BroadlinkRMPlatform.addAccessories (/usr/lib/node_modules/homebridge-broadlink-rm/platform.js:74:24)
мар 21 00:41:20 homekit homebridge[11645]:     at BroadlinkRMPlatform.accessories (/usr/lib/node_modules/homebridge-broadlink-rm/node_modules/homebridge-platform-helper/platform.js:27:10)
мар 21 00:41:20 homekit homebridge[11645]:     at Server._loadPlatformAccessories (/usr/lib/node_modules/homebridge/lib/server.js:364:20)
мар 21 00:41:20 homekit homebridge[11645]:     at Server._loadPlatforms (/usr/lib/node_modules/homebridge/lib/server.js:308:16)
мар 21 00:41:20 homekit homebridge[11645]:     at Server.run (/usr/lib/node_modules/homebridge/lib/server.js:85:36)
мар 21 00:41:20 homekit homebridge[11645]:     at module.exports (/usr/lib/node_modules/homebridge/lib/cli.js:40:10)
мар 21 00:41:20 homekit homebridge[11645]:     at Object.<anonymous> (/usr/lib/node_modules/homebridge/bin/homebridge:17:22)
мар 21 00:41:20 homekit homebridge[11645]:     at Module._compile (module.js:649:30)
мар 21 00:41:20 homekit homebridge[11645]:     at Object.Module._extensions..js (module.js:660:10)
мар 21 00:41:20 homekit homebridge[11645]:     at Module.load (module.js:561:32)
мар 21 00:41:20 homekit homebridge[11645]:     at tryModuleLoad (module.js:501:12)
мар 21 00:41:20 homekit homebridge[11645]:     at Function.Module._load (module.js:493:3)
мар 21 00:41:20 homekit homebridge[11645]:     at Function.Module.runMain (module.js:690:10)
мар 21 00:41:20 homekit homebridge[11645]:     at startup (bootstrap_node.js:194:16)
мар 21 00:41:20 homekit homebridge[11645]:     at bootstrap_node.js:666:3```

**Comment by cubilon** [comment]
yes, I noticed the same

**Comment by lprhodes** [comment]
I've fixed this issue in 3.3.0 and also added coloured `[DEBUG]` text to the beginning to relevant lines.
